x-pack/solutions/security/test/osquery_cypress/runner_qa.ts

```diff
@@ -7,7 +7,7 @@
 
 import { prefixedOutputLogger } from '@kbn/security-solution-plugin/scripts/endpoint/common/utils';
 import { createToolingLogger } from '@kbn/security-solution-plugin/common/endpoint/data_loaders/utils';
-import { setupStackServicesUsingCypressConfig } from '@kbn/security-solution-plugin/public/management/cypress/support/common';
+import { setupStackServicesUsingCypressConfig } from '@kbn/cypress-test-helper/src/services/common';
```

Comment:
I had to move it under `@kbn/cypress-test-helper` platform package to avoid circular dependency between test-helper packages.